### ISSUE TITLE
Hydra ML config manager instead of Argparse

### ISSUE THREAD

**Issue by mauriciogtec** [NSAPH-Projects/topological-equivariant-networks#18]
Use [Hydra](https://hydra.cc) instead of Argparse, since it is easier to manage configs of different models, datasets, etc.

For example, iinstead of instantiating models with lengthy code, such as:

```python
if args.model == "....":
   model = ....
```

We can use a single call 

```python
model = instantiate(cfg.model)
```

**Comment by mauriciogtec** [comment]
currently used by the etnn_spatial

**Comment by audiracmichelle** [comment]
Closed in [77bf3ba](https://github.com/NSAPH-Projects/topological-equivariant-networks/commit/77bf3ba74203c380b42ea98bbd2a9e91052cb5bf)
